Cartfile.private

```diff
@@ -0,0 +1 @@
+github "AliSoftware/OHHTTPStubs" "8.0.0"
```

Comment:
`~> 7.0` here.

Reply:
I've changed it to `~> 7.0` because it was when they updated to Swift 4.2, otherwise I will not be able to build the project with a Swift 5 project. Is that ok?

Reply:
:+1: